__tests__/subscriptions.service.test.ts

```diff
@@ -21,21 +21,24 @@ jest.mock('../src/service/pubsub', (): any => ({
   },
 }));
 
-jest.mock('./pubsub/subscriptions/example.subscription', (): any => ({
-  __esModule: true,
-  default: jest.fn().mockImplementation((): any => ({
-    getTopicName: (): any => {
-      return 'test-topic';
-    },
-    setMongooseConnection: function (): any {
-      return this;
-    },
-    init: function (): any {
-      return this;
-    },
-    start: jest.fn(),
-  })),
-}));
+jest.mock(
+  './pubsub/subscriptions/test-topic.example.subscription',
+  (): any => ({
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8106467)
```suggestion
  (): unknown => ({
```